src/MLOps.NET.SQLServer/Migrations/20201001214039_AddPackageDependency.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace MLOps.NET.SQLServer.Migrations
+{
+    /// <summary>
+    /// 
```

Comment:
Should these summaries be empty?

Reply:
This particular ones are just for the migrations, and as the solution as a whole enforces comments on public classes they are required but not really needed...so left them empty :)